netbox/netbox/views/generic.py

```diff
@@ -689,15 +692,18 @@ def get(self, request):
     def post(self, request):
         logger = logging.getLogger('netbox.views.BulkImportView')
         new_objs = []
-        form = self._import_form(request.POST)
+        form = self._import_form(request.POST, request.FILES)
 
         if form.is_valid():
             logger.debug("Form validation was successful")
 
             try:
                 # Iterate through CSV data and bind each row to a new model form instance.
                 with transaction.atomic():
-                    headers, records = form.cleaned_data['csv']
+                    if request.FILES:
```

Comment:
We should add a validation method to the form to check for both `csv` and `upload_csv` set, and raise a ValidationError if so as a sanity check.

Reply:
Added a method for this within the _import_form definition